nautobot/extras/models/mixins.py

```diff
@@ -180,6 +180,9 @@ class Meta:
 
     @property
     def static_groups(self):
+        """
+        Returns a QuerySet of StaticGroups that have this object as a member.
```

Comment:
```suggestion
        Returns a queryset of StaticGroups that have this object as a member.
```

Reply:
But it is an instance of `QuerySet` class.